release notes/v1.0.0.md

```diff
@@ -0,0 +1,139 @@
+   /  \/    \    | |/ /  /   ‾‾\
+  /          \   |   (  |  (‾)  |
+ / __________ \  |_|\_\  \_____/
+
+     execution: local
+        script: script.js
+        output: -
+
+     scenarios: (100.00%) 1 scenario, 1 max VUs, 10m30s max duration (incl. graceful stop):
+              * default: 10 iterations shared among 1 VUs (maxDuration: 10m0s, gracefulStop: 30s)
+
+INFO[0000] iteration 0 executed         source=console
+INFO[0000] iteration 1 executed         source=console
+INFO[0000] iteration 2 executed         source=console
+ERRO[0000] iteration 3: marked the test as failed
+INFO[0000] iteration 3 executed         source=console
+INFO[0001] iteration 4 executed         source=console
+INFO[0001] iteration 5 executed         source=console
+INFO[0001] iteration 6 executed         source=console
+INFO[0001] iteration 7 executed         source=console
+INFO[0001] iteration 8 executed         source=console
+INFO[0001] iteration 9 executed         source=console
+
+
+  █ TOTAL RESULTS
+
+    HTTP
+    http_req_duration.......................................................: avg=129.22ms min=127.9ms  med=129.17ms max=130.83ms p(90)=130.51ms p(95)=130.67ms
+      { expected_response:true }............................................: avg=129.22ms min=127.9ms  med=129.17ms max=130.83ms p(90)=130.51ms p(95)=130.67ms
+    http_req_failed.........................................................: 0.00%  0 out of 10
+    http_reqs...............................................................: 10     5.763858/s
+
+    EXECUTION
+    iteration_duration......................................................: avg=173.46ms min=128.07ms med=129.43ms max=569.82ms p(90)=174.89ms p(95)=372.36ms
+    iterations..............................................................: 10     5.763858/s
+    vus.....................................................................: 1      min=1       max=1
+    vus_max.................................................................: 1      min=1       max=1
+
+    NETWORK
+    data_received...........................................................: 32 kB  19 kB/s
+    data_sent...............................................................: 1.0 kB 601 B/s
+
+running (00m01.7s), 0/1 VUs, 10 complete and 0 interrupted iterations
+default ✓ [======================================] 1 VUs  00m01.7s/10m0s  10/10 shared iters
+ERRO[0001] test run was marked as failed
+k6 exited with status: 110
+```
+
+</details>
+
+## UX improvements and enhancements
+
+- [#4698](https://github.com/grafana/k6/pull/4698) Displays threshold values even when are not configured in `summaryTrendStats` option.
+- [#4699](https://github.com/grafana/k6/pull/4699) Drops the link of the legacy k6 website from the user agent.
+
+## Bug fixes
+
+- [#4717](https://github.com/grafana/k6/pull/4717) Safeguards against `pressedKeys` being updated concurrently in browser module. Also empty struct instead of a bool for a set in browser module.
```

Comment:
I don't think the last sentence adds anything useful to the fix title. It's an implementation detail.

```suggestion
- [#4717](https://github.com/grafana/k6/pull/4717) Safeguards against `pressedKeys` being updated concurrently in browser module.
```